alex_questions.md

```diff
@@ -0,0 +1,19 @@
+# direct TX
+
+- What format can we expect for the direct TX on the DA? some wrapper type would be useful to unpack and distinct from
+  random bytes
+- Should we always include direct TX although they may be duplicates to mempool TX? spike: yes, first one wins.
+- Should we fill the block space with directTX if possible or reserve space for mempool TX
+- What should we do when a sequencer was not able to add a direct-TX within the time window?
+- Do we need to trace the TX source chanel? There is currently no way to find out if a sequencer adds a direct TX "to
+  early". Can be a duplicate from mempool as well. 
+- What is a good max size for a sequencer/fallback block? Should this be configurable or genesis?
+- Do we need to track the sequencer DA activity? When it goes down with no direct TX on DA there are no state changes.
```

Comment:
Yes, in order to decide when to enter fallback mode, full nodes need to assess the liveness of the sequencer. The liveness of the sequencer is dependent on whether they're posting evolve blocks on the DA layer. 

Gossiping evolve blocks over P2P shouldn't be considered the way to check liveness since they're vulnerable to P2P split network attacks like eclipse attacks. Opting to listen over P2P is purely optional for a full node. So we should assume the case for a full node only listening to DA for evolve blocks